apps/prairielearn/elements/pl-string-input/pl-string-input.mustache

```diff
@@ -1,66 +1,125 @@
+<script>
+    $(function(){
+        document.querySelectorAll('.multiline-input#pl-string-input-{{uuid}}-area').forEach((input) => {
+            input.addEventListener('input', function () {
+            // Adjusts the height based on the feedback content. If the feedback changes, the height
+            // changes as well. This is done by resetting the height (so the scrollHeight is computed
+            // based on the minimum height) and then using the scrollHeight plus padding as the new height.
+            this.style.height = '';
+            if (this.scrollHeight) {
+                const style = window.getComputedStyle(this);
+                this.style.height =
+                this.scrollHeight + parseFloat(style.paddingTop) + parseFloat(style.paddingBottom) + 'px';
+            }
+            });
+            input.dispatchEvent(new Event('input'));
+        });
+    });
+</script>
+
 {{#question}}
+
 {{#inline}}<span class="form-inline d-inline-block ml-2">{{/inline}}
-    <span class="input-group pl-string-input">
+    {{#multiline}}
         {{#label}}
-        <span class="input-group-prepend">
-            <span class="input-group-text" id="pl-string-input-{{uuid}}-label">{{{label}}}</span>
-        </span>
+        <label
+            id="pl-string-input-{{uuid}}-label"
```

Comment:
I believe you don't need an ID on the label since it isn't being referenced by `aria-labelledby` or the like.

Reply:
https://github.com/PrairieLearn/PrairieLearn/pull/9438/commits/034e1035f27db64dbf47102accab254a7e002645